packages/menu/src/MenuItem.ts

```diff
@@ -570,38 +592,37 @@ export class MenuItem extends LikeAnchor(
     }
 
     protected async handleSubmenuPointerleave(): Promise<void> {
-        requestAnimationFrame(() => {
-            this.recentlyLeftChild = false;
-        });
+        this.recentlyLeftChild = false;
```

Comment:
Does removing the requestAnimationFrame here cause the state to be updated more immediately? What the goal of the animation frame being present initially?

Reply:
@castastrophe it was causing a delay that left two menu items with hover styling.